data/virtualization/autoyast/host_15.xml.ep

```diff
@@ -358,49 +358,18 @@ cat >> /root/.config/fontconfig/fonts.conf <<EOF
   </alias>
 </fontconfig>
 EOF
+]]>
+        </source>
+      </script>
+      <script>
+        <filename>config_ssh.sh</filename>
+        <source><![CDATA[
 mkdir -p -m 700 /root/.ssh
 cat >> /root/.ssh/id_rsa<< EOF
------BEGIN OPENSSH PRIVATE KEY-----
-b3BlbnNzaC1rZXktdjEAAAAABG5vbmUAAAAEbm9uZQAAAAAAAAABAAABlwAAAAdzc2gtcn
-NhAAAAAwEAAQAAAYEAtFzEfa/DTa+fPvAZmw/gsk0HfYkKoiGI2OOYFYhKpdPkea2oVh0A
-cbkgMZCWzzC7ncptX33Ut/jg7gVTvJxwAWe3hmj2SUEEztGNm6SVworP9BsT1A/eBCfjSP
-+P6s6crsOQm3ztjGs13hd/JgUTv3HmeeGSJGTLqkx6uZKfq4atislAcbpvbVgu7h4dCRQq
-Ea6NvLkUoDZ97lUjHtaQIoXasJWegmg8Hzf9/mhKg08/3vxONQCiYw8ew2gjumpSm5J9lf
-A9mpbYY5we2bSjbGH5IkOlfarvZRLYHhUHhGSZJVj26vgmC3L6+xcrPDOegADagc2uSnPG
-uwC2WTRG6225rAsvmr0vz7X7ey1zYw3zU1F0PjUQzR7b3PVJhzsfXHaJIFOeIcQH8b/Ctr
-HL2lngUCRwLu+qDTK5xsSiDHTQR9GWzbnUoC5R1fWfzyGGFze18R3CbiIPYY6Nq8lHT3UQ
-OChmnmTfa1569PPKiVLQyY9DAWkUgmbauc1miclJAAAFiG6RBSlukQUpAAAAB3NzaC1yc2
-EAAAGBALRcxH2vw02vnz7wGZsP4LJNB32JCqIhiNjjmBWISqXT5HmtqFYdAHG5IDGQls8w
-u53KbV991Lf44O4FU7yccAFnt4Zo9klBBM7RjZuklcKKz/QbE9QP3gQn40j/j+rOnK7DkJ
-t87YxrNd4XfyYFE79x5nnhkiRky6pMermSn6uGrYrJQHG6b21YLu4eHQkUKhGujby5FKA2
-fe5VIx7WkCKF2rCVnoJoPB83/f5oSoNPP978TjUAomMPHsNoI7pqUpuSfZXwPZqW2GOcHt
-m0o2xh+SJDpX2q72US2B4VB4RkmSVY9ur4Jgty+vsXKzwznoAA2oHNrkpzxrsAtlk0Rutt
-uawLL5q9L8+1+3stc2MN81NRdD41EM0e29z1SYc7H1x2iSBTniHEB/G/wraxy9pZ4FAkcC
-7vqg0yucbEogx00EfRls251KAuUdX1n88hhhc3tfEdwm4iD2GOjavJR091EDgoZp5k32te
-evTzyolS0MmPQwFpFIJm2rnNZonJSQAAAAMBAAEAAAGBAKhduOcDPjO079kW1TBVABIxqf
-5cAVscJt0giIYBNn3acXvMykmoxRNkF1Ntf/plqZ5AqxzrH7mlUIOg4Ww+NKh7I20Lam0z
-jsNqBuD2IP78Cef7puTc8wm6Goe4WaZ9vPG/iaw8UJw2MJDkKkNZlfeu4dGA6qWimiSdRC
-sbXoYGMNZPzCLeQMo3+Yc7ASvKcQMUiSdVNpXgiGoFe8V70g0IGv+gi9l8aDNUV3w36ubt
-Adisem0r7GrAYJ1VB5UrTeOtMB0jfUfhTWpzskPBIj1no6k0iv+3/xX1Q/A33zgo62qNlY
-8ufw1E45dH/e/V2EXrNt/muP5l12Jm/oUggLepkPfXvDOmjO6HUn0MrSIRQc+h/jHbW8oM
-uB/9w84blDtAxPBFlWBwqwFMiwBrcb69wX+hSSk4lt1fgb0cmVdZ/g6/LUZ+rhDJBoF8tt
-hLQNwc/spI2jyVWQA6xI0QCyXnAZkhDl7SG4l1+bNSbdVOycTwJIwxtFY83c/TOOGFiQAA
-AMEAvcP/w/VVhC9dUeolArTs+G+NLIhRAmAtF1buYqfpyKw+XFKo3ZCDV8L/YeApQRiggW
-UN3fRdFmduW6y7gGQFhmjZsPDzjBGnSAYHX7LP5JaykaV/kMzXKPLPQf5JvAMPw6mokTd8
-yHuwPczjlzAkAXeXNk/dJ3kk5y0CL3lGC5KTLLlSSrhXlZai82SH1QNuudZLKsHUZ8du+f
-Qjh/pwD6+DDT2mg2kxKA12oPlmTOOUnI7cQxT7HQEYjybK9CHXAAAAwQDnvXWMmZKgWfnc
-4LZlxCYQvr5ZXQUksxgACW9WWoBR/7e9SjQs1th8NmEbX5JQsgxuXjXLhalr30UZzTbO82
-u0lwyCraVgAfr2YL8PuhCkSimi6NI9CrvetfbgHPsDAR45bMmLDbU3kGFKN47T2v6oYSYX
-KY6dS805LzaQH2+RnHR+xxZjd305lAZXkxZT54MOV80L3YSk7HyfCdY0WUS01UBD6uAQbw
-cvmWDxCp8gUkKrFL/ASyV9yK+IYy4ZJAcAAADBAMc+aBhA+T9Jwq7/x6lCzrG4zO3w+l0X
-AztXQAAHH64nkiECQ+pTn/qHH6TEfSkeqj97o/RDlGbf3s0NogUZEhSxGMxMhoqPatMa93
-UajDxU+4Xchxtt4S6MPHMIYRRcRi+pqa91Iz8wYYsN3PHVoRKsbRI2H0Nh5cqjjHeN1DhD
-lbzZvl0rkSQSotKBVWnX/1aU5Pk0RX8HKzYyLZiTjXJAX8RKnem5gsmeVv6yRbcQ2Fp5+I
-t3XODgiOxVK190LwAAAApyb290QHJpdmVyAQIDBAUGBw==
------END OPENSSH PRIVATE KEY-----
+<%= $get_var->('ID_RSA') %>
```

Comment:
Hi Roy,
Thanks for the quick PR! Super cool!

Just some minor comment:
- for the rsa key, what's the current length? Please use >1024, eg 4096. This is new requirement for 15sp6. 
 You can get your key length by cmd like below
```alice:/home/alice/Documents/github_repo/qa-automation/libs/qa_lib_keys # ssh-keygen -lf qa_lib_keys/id_rsa.pub
4096 SHA256:uZvTO8OP666BqThpABfXwvuPs0uHmBigm8PM2DA6YUI root@qadb2 (RSA)
```

- For setting name of the keys, suggest to have prefix `_SECRET`, which will be hided in openqa jobs( log files, vars...), so more secure. Eg `_SECRET_RSA_PRIV_KEY`, `_SECRET_RSA_PUB_KEY`

Reply:
hansolo:~/.ssh # ssh-keygen -lf id_rsa
3072 SHA256:2Q9+4OlQZAiROq4iJ52ez3DDuJqWYX+cnjYuxLaki1E id_rsa.pub (RSA)
> 1024. it is ok for sles15sp6, I think.

Reply:
Updated setting name of the keys to _SECRET_RSA_PRIV_KEY and _SECRET_RSA_PUB_KEY